CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add new command: [`expand`] [#964]
+- Add `--drop_axiom_annotation` option to drop axiom annotations in [`remove`] and [`filter`] [#1023]
```

Comment:
Always kebap case for robot long options; https://www.theserverside.com/definition/Kebab-case

Reply:
Fixed